### ISSUE TITLE
Performance Regression?

### ISSUE THREAD

**Issue by valleyman86** [groue/GRDB.swift#249]
So I was running tests with SQLite.swift, GRDB and Core Data and I am finding that Core Data and GRDB are pretty close is speed for my use case (simple table with string and 2 ints no uniques)

So I went and ran the test in GRDB. It seems to me that GRDB is considerably slower than SQLite.swift. My data can be built in SQLite.swift in about 22 seconds while GRDB is running around 33.

Based on the test below SQLite.swift is close to what is expected but GRDB is not based on the post here. https://github.com/groue/GRDB.swift/wiki/Performance

[GRDBOSXPerformanceTests.InsertPositionalValuesTests testSQLite]' measured [Time, seconds] average: 0.027

[GRDBOSXPerformanceTests.InsertPositionalValuesTests testGRDB]' measured [Time, seconds] average: 0.575,

[GRDBOSXPerformanceTests.InsertPositionalValuesTests testSQLiteSwift]' measured [Time, seconds] average: 0.160,

**Comment by groue** [comment]
Hello @valleyman86.

Can you tell which version of the library you are measuring? And if you run tests in the Release configuration?

**Comment by valleyman86** [comment]
Build is 1.3.0 (master branch) and I actually did try just downloading it directly (should be 1.3.0) which is where I initially saw the issues.  

I tried in release and I see that it now runs very close to SQLite now. Seems to be on my end and is actually faster than SQLite.swift in release. Thanks!

[GRDBOSXPerformanceTests.InsertPositionalValuesTests testGRDB]' measured [Time, seconds] average: 0.066

[GRDBOSXPerformanceTests.InsertPositionalValuesTests testSQLiteSwift]' measured [Time, seconds] average: 0.096
